gradle/lib/dependabot/gradle/version.rb

```diff
@@ -132,7 +132,7 @@ def convert_dates(version, other_version)
       end
 
       def split_into_prefixed_tokens(version)
-        ".#{version}".split(/(?=[\-\.])/)
+        ".#{version}".tr("_", "-").split(/(?=[\-\.])/)
```

Comment:
I think we could just include it in the regex as one of the values to split on, instead of converting to a dash and then matching on that later on?

```suggestion
        ".#{version}".split(/(?=[\-\.\_])/)
```

Reply:
I did that in the last commit. It requires another change in the file.

Reply:
Yep, that makes sense to me, thanks for making the change!